src/collar/oc_settings.lsl

```diff
@@ -15,6 +15,90 @@ https://github.com/OpenCollarTeam/OpenCollar
 
 //string g_sParentMenu = "Apps";
 
+
+integer g_iSettingsLoaded;
+integer g_iLastStride;
+string  g_sVariableView;
+string  g_sTokenView="";
+
+// return a CSV containing all global_var=val
+string SettingsFor(string SelectToken){
+    list lSettings;    
+    integer i = llGetListLength(g_lSettings); 
+    while((i-=2)>=0){        
+        list lTmp = llParseString2List(llList2String(g_lSettings,i),["_"],[]);
+        string sTok = llList2String(lTmp,0);
+        string sVar = llList2String(lTmp,1);
+        if(sTok == SelectToken)
+            lSettings+=llList2String(g_lSettings,i)+"="+llList2String(g_lSettings,i+1);
+    }
+    return llList2CSV(lSettings);
+}
+
+// sends "ALIVE oc_setttings <global_var=val list>"
+sendALIVE(){
```

Comment:
Can you capitalize this consistently with the other functions?

```suggestion
SendALIVE(){
```

Reply:
done